content/docs/projects/approver-policy.md

```diff
@@ -298,7 +298,7 @@ spec:
   ...
   selector:
     issuerRef:
-    - name: "my-ca"
+      name: "my-ca"
       kind: "*Issuer"
       group: "cert-manager.io"
```

Comment:
We don't yet publish the API docs for open-source approver-policy, but in the [Jetstack documentation](https://platform.jetstack.io/documentation/reference/approver-policy-enterprise/kubernetes-api#certificaterequestpolicyspecselector), 
you can see that this issuerRef should be an object rather than a list